mingw-w64-sundials/PKGBUILD

```
@@ -66,7 +69,6 @@ build() {
     -D BUILD_ARKODE=ON \
     -D BUILD_CVODE=ON \
     -D BUILD_CVODES=ON \
-    -D BUILD_FORTRAN77_INTERFACE=ON \
     -D BUILD_FORTRAN_MODULE_INTERFACE=OFF \
```

```suggestion
    -D BUILD_FORTRAN_MODULE_INTERFACE=ON \
```
Since FORTRAN77 is dropped, Fortran 2003 should be enabled

---

Unfortunately we won't be able to do that because their Fortran 2003 interface requires 64-bit indexing which we cannot enable because the dependencies are built with 32-bit indexing.

---

> Unfortunately we won't be able to do that because their Fortran 2003 interface requires 64-bit indexing which we cannot enable because the dependencies are built with 32-bit indexing.

I have taken a look at their Fortran 2003 Interfaces, They are using `integer(C_INT)` in their declarations and this shouldn't be a problem at all.
I couldn't find where do they use 64-bit indexing, Could you give me source location (in https://github.com/LLNL/sundials)

---

I see now https://github.com/LLNL/sundials/blob/master/src/ida/fmod/fida_mod.f90#L829

---

Right, and then we're also explicitly blocked by the [build system](https://github.com/LLNL/sundials/blob/master/cmake/SundialsBuildOptionsPre.cmake#L161) as well as a [critical source file](https://github.com/LLNL/sundials/blob/c0097c0bfd56e8f94dbb588e0894fdceacf6f737/src/sundials/fmod/fsundials_types_mod.c#L216). So it doesn't seem like they plan on supporting it any time soon.